package.json

```diff
@@ -55,12 +57,15 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
-    "msw": "1.2.3",
+    "msw": "2.0.13",
     "npm-run-all": "4.1.5",
     "prop-types": "15.8.1",
     "stylelint": "13.13.1",
     "stylelint-config-recommended-scss": "4.3.0",
     "stylelint-scss": "3.21.0",
+    "ts-patch": "^3.1.1",
+    "typescript": "^5.3.3",
+    "undici": "^5.0.0",
```

Comment:
needs to be pin to version 5.x, there's an issue with axios and msw - https://github.com/mswjs/msw/discussions/1915